c2corg_api/search/sync.py

```diff
@@ -62,13 +64,27 @@ def remove_doc():
             # https://github.com/c2corg/v6_api/issues/89
             DBSession.refresh(document)
 
+        is_user_profile = isinstance(document, UserProfile)
+        if is_user_profile:
+            # set user login + full-name as document title so that it can
+            # be searched
+            user_login, user_name = DBSession. \
+                query(User.name, User.username). \
+                filter(User.id == document.document_id). \
+                first()
+            user_title = '{0} {1}'.format(user_name or '', user_login or '')
```

Comment:
Which pseudo fields are secret or not is a bit complicated and seems to have changed over time.
Is the pseudo visible in v6? If no, then why should it be indexed in Elasticsearch?

For Discourse we may use the more or less secret pseudo and allow user to change it afterwards.
See https://github.com/c2corg/v6_forum/issues/8.